src/CoreBundle/private/library/classes/dbobjects/TCMSTableConf.class.php

```diff
@@ -490,23 +490,29 @@ public function ExportTable()
     }
 
     /**
-     * fetches the TdbFooBaa class name and loads it for current table config record.
+     * @param string|null $id - the id of the record to load, if any
      *
-     * @param string|null $recordID - record to load
-     *
-     * @return TCMSRecord
+     * @return TCMSRecord|null - null if the class does not exist or the loading of the id fails if given
      */
-    public function GetTableObjectInstance($recordID = null)
+    public function GetTableObjectInstance($id = null)
```

Comment:
why are we changing this method in this context? Also: we are changing the signature here, which we should only do in a minor update.

Reply:
Changes to the method:

Two checks (eg class_exists) were added. Otherwise there will be (non-prohibitable) crashes in it.

I revereted the renaming of the parameter. However that would not have had any (breaking) consequences.

Reply:
However you are right that it is not really related here...